src/content/guides/farmlands-portal-introduction.mdx

```diff
@@ -193,6 +193,58 @@ On the Settings page, you can change your business information. Please note this
 
 ![Centrapay business portal settings page](/centrapay-portal-settings.png)
 
+## FAQs
+
+- **What do we do if a customer returns one or more items, and the invoice has already been sent to Farmlands?**
```

Comment:
@IsaacMurtagh FYI that not using a heading means we won't be able to link readers directly to specific questions (Astro will not generate anchors for them)

Reply:
Yep, but the sidebar grows very large if they're headings because the questions are quite long.